test/build_headers/sdk/km/c/csdkheaders.vcxproj

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <ClCompile Include="sdkheaders.c" />
+  </ItemGroup>
+  <PropertyGroup>
+    <ProjectGuid>{3ddfcc4f-61ed-42f2-9a0b-94e1fdea6146}</ProjectGuid>
+    <TemplateGuid>{0a049372-4c4d-4ea0-a64e-dc6ad88ceca1}</TemplateGuid>
```

Comment:
these are old copy/pastes - no longer any need for TemplateGuid in any of our vcxproj files